fabfile/add.py

```diff
@@ -66,9 +63,8 @@ def remote_server():
         print "{0} : {1}, ".format(attr, value)
     print "}\n"
     raw_input("Press Enter to Continue or Ctrl+C to Cancel")
-    with bash():
-        local("cp -rf $FAB_PATH/../templates ./templates.tmp")
-        recursive_file_modify("./templates.tmp", settings)
+    local("cp -rf ./fabfile/../templates ./templates.tmp")
```

Comment:
isn't `./fabfile/../templates` the same as `./templates`? Is that the right path?